.github/workflows/helm.yml

```diff
@@ -0,0 +1,19 @@
+name: Update Helm Chart POMI version
+on:
+  release:
+    types: [released]
+
+jobs:
+  create_helm_chart_pr:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Create PR using Version Bump 
+      run: |
+        appVersion=${{github.event.release.tag_name}}
+        chartName=nri-prometheus
+        
+        curl -H "Accept: application/vnd.github.everest-preview+json" \
+        -H "Authorization: token ${{ secrets.VERSION_BUMP_TOKEN }}" \
```

Comment:
This is probably good as it is, but I wanted to double check: Would this work with `secrets.GITHUB_TOKEN`, which is automatically created by github actions? Maybe it won't if it's a different repo, but might be worth saving yet another secret to keep track of.

Reply:
I worked on this one and sadly it needs a Personal token kind :( . [More details](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows)